app/controllers/state_file/questions/nj_ineligible_property_tax_controller.rb

```diff
@@ -3,8 +3,66 @@ module Questions
     class NjIneligiblePropertyTaxController < QuestionsController
       include ReturnToReviewConcern
 
-      def self.show?(intake)
-        intake.household_rent_own_neither?
+      helper_method :ineligible_reason
+      helper_method :on_home_or_rental
+
+      def determine_reason
+        if current_intake.household_rent_own_neither?
+          return "neither"
+        end
+
+        if params[:on_home_or_rental] == "home"
+          if current_intake.homeowner_home_subject_to_property_taxes_no?
+            return "property_taxes"
+          elsif current_intake.homeowner_main_home_multi_unit_yes? && current_intake.homeowner_main_home_multi_unit_max_four_one_commercial_no?
+            return "multi_unit_conditions"
+          end
+        end
+
+        if params[:on_home_or_rental] == "rental"
+          if current_intake.tenant_home_subject_to_property_taxes_no?
+            return "property_taxes"
+          elsif current_intake.tenant_building_multi_unit_yes? && current_intake.tenant_access_kitchen_bath_no?
+            return "multi_unit_conditions"
+          end
+        end
+      end
```

Comment:
[sand] - (mentioned on live review) since `determine_reason()` is dependent on non-view logic, should probably live in `NjPropertyTaxEligibility` so there is one source of truth

Reply:
Captured in follow-up ticket https://github.com/newjersey/affordability-pm/issues/227